src/sysusers/sysusers.c

```diff
@@ -1787,7 +1795,11 @@ int main(int argc, char *argv[]) {
                 int j;
 
                 for (j = optind; j < argc; j++) {
-                        k = read_config_file(argv[j], false);
+                        if (arg_direct)
+                                /* Use (argument):n, where n==1 for the first positional arg */
+                                k = parse_line("(argument)", j - optind + 1, argv[j]);
+                        else
+                                k = read_config_file(argv[j], false);
```

Comment:
i am tempted to say that if we add this here, we should probably add the equivalent to to systemd-tmpfiles and systemctl-sysctl right-away, too. Or at least TODO list items, so that we remember to add this.

Reply:
I'll start working on this, *once* we settle on a logic. I don't want to redo all the variations three times ;)